Azure.ResourceManager.Core/ResourceGroupContainer.cs

```diff
@@ -100,6 +101,7 @@ public Pageable<ResourceGroup> List(CancellationToken cancellationToken = defaul
         /// List the resource groups for this subscription.
         /// </summary>
         /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="P:System.Threading.CancellationToken.None" />. </param>
+        /// <returns> A response with the <see cref="AsyncPageable{ResourceGroup}"/> operation for this resource. </returns>
```

Comment:
use the standard response text for asyncpageable
https://github.com/m-nash/azure-proto-sdk/blob/master/XmlDocStandard.md#asyncpageable